minvws/nl-kat-coordination
# make some things look better

### Changes
- Added `horizontal-scroll` div wrapped around some tables.
- Plugins table: added expando row for plugin type and description. this makes the table more compact.
- Remove padding for final report, it already have a max-width

### Demo

Major change:

Paddings final report
<img width="1671" alt="image" src="https://github.com/user-attachments/assets/72e744ea-9c8a-4a4d-a409-2144ee52fc1a">


FROM THIS
<img width="1041" alt="image" src="https://github.com/user-attachments/assets/db8898d1-834a-482d-a993-1991c99799cf">

TO THIS:
<img width="965" alt="image" src="https://github.com/user-attachments/assets/6d94bf0c-13bc-49d1-bcc8-ee25f3d2d8c4">

Mobile now is fixed for horizontal scroliing:
<img width="475" alt="image" src="https://github.com/user-attachments/assets/bc845055-9274-4172-be9f-cf7d35cdc612">

### QA notes

You need to also run yarn dev or yarn build to see padding changes on the final report.

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
